Source/Core/Core/BootManager.cpp

```diff
@@ -224,7 +224,6 @@ bool BootCore(const std::string& filename, SConfig::EBootBS2 type)
 
   StartUp.m_BootType = SConfig::BOOT_ISO;
   StartUp.m_strFilename = filename;
-  StartUp.m_LastFilename = filename;
   StartUp.SaveSettings();
```

Comment:
Is there a reason for saving the settings other than making sure that LastFilename is up to date?